llvm/lib/Transforms/Scalar/LoopIdiomRecognize.cpp

```diff
@@ -294,6 +294,10 @@ bool LoopIdiomRecognize::runOnLoop(Loop *L) {
   if (Name == "memset" || Name == "memcpy")
     return false;
 
+  // Prevent from asan interception in kernel
+  if (Name == "__asan_set_shadow_local_memory")
```

Comment:
I see there is a while loop inside __asan_set_shadow_local_memory. What form would LoopIdiomRecognize transform the loop to?

Reply:
it will transform the loop into "memset", which will be intercepted by asan, and does unexpected memory access checks and aborts the program.